test/pom.xml

```diff
@@ -34,6 +34,22 @@
          <groupId>junit</groupId>
          <artifactId>junit</artifactId>
       </dependency>
+	   <dependency>
+		   <groupId>org.junit.jupiter</groupId>
+		   <artifactId>junit-jupiter-api</artifactId>
+	   </dependency>
+	   <dependency>
+		   <groupId>org.junit.jupiter</groupId>
+		   <artifactId>junit-jupiter-engine</artifactId>
+	   </dependency>
+	   <dependency>
+		   <groupId>org.junit.vintage</groupId>
+		   <artifactId>junit-vintage-engine</artifactId>
+	   </dependency>
+	   <dependency>
```

Comment:
please group the mockito-junit with the mockito core dependency. make sure that is the case in all pom.xml . sorry that I did not see this earlier